write-fonts/src/tables.rs

```diff
@@ -1,6 +1,8 @@
 //! A directory of all the font tables.
 
 pub mod base;
+// TODO fix bug with enum defaults
+//pub mod cmap;
```

Comment:
I doubt we want to commit this?

Reply:
No, but we should definitely fix that bug!